src/Microsoft.AspNetCore.Server.Testing/Deployers/IISExpressDeployer.cs

```diff
@@ -34,21 +34,23 @@ public override DeploymentResult Deploy()
                 DotnetPublish();
             }
 
+            var webRootLocation = DeploymentParameters.PublishApplicationBeforeDeployment ? DeploymentParameters.PublishedApplicationRootPath : DeploymentParameters.ApplicationPath;
```

Comment:
If you make it appRoot then StartIISExpress is correct, but DeploymentResult.WebRootLocation is wrong. What's WebRootLocation used for?